src/with_clause/create_materialized_view_with_clause.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include <catalog/pg_type.h>
+#include <fmgr.h>
+#include <nodes/makefuncs.h>
+
+#include "compat/compat.h"
+
+#include "alter_table_with_clause.h"
+#include "create_materialized_view_with_clause.h"
+#include "cross_module_fn.h"
+#include "ts_catalog/continuous_agg.h"
+#include "with_clause_parser.h"
+
+static const WithClauseDefinition continuous_aggregate_with_clause_def[] = {
+    [CreateMaterializedViewFlagContinuous] = {
+        .arg_names = {"continuous", NULL},
+        .type_id = BOOLOID,
+        .default_val = (Datum)false,
+    },
+    [CreateMaterializedViewFlagCreateGroupIndexes] = {
+        .arg_names = {"create_group_indexes", NULL},
+        .type_id = BOOLOID,
+        .default_val = (Datum)true,
+    },
+    [CreateMaterializedViewFlagMaterializedOnly] = {
+        .arg_names = {"materialized_only", NULL},
+        .type_id = BOOLOID,
+        .default_val = (Datum)true,
+    },
+    [CreateMaterializedViewFlagCompress] = {
+        .arg_names = {"enable_columnstore", "compress", NULL},
```

Comment:
```suggestion
        .arg_names = {"columnstore", "enable_columnstore", "compress", NULL},
```

Reply:
I think this should be a separated PR including the proper regression tests.

Reply:
for normal compression this is already an alias so we should keep that in sync, ideally we can have them share the option definitions but we can do that in followup